document/core/valid/instructions.rst

```diff
@@ -154,6 +154,325 @@ Numeric Instructions
    }
 
 
+.. index:: simd instruction
+   pair: validation; instruction
+   single: abstract syntax; instruction
+
+.. _valid-instr-simd:
+
+SIMD Instructions
+~~~~~~~~~~~~~~~~~
+
+
+.. todo::
+   do we need v128.const here again? It's covered above by Numeric Instructions.
+
+.. todo::
+   can we define these instructions that begin with "v128." together with the numerics? The structure, "t.op" matches up. This will require us to specify in the conventions section of syntax/instructions to extend \binop, rather than define a new \vsbinop
+
+.. _aux-bool:
+
+SIMD instructions can have a prefix to describe the :ref:`shape <syntax-simd-shape>` of the operand. Packed numeric types, |i8| and |i16|, are not :ref:`value type <syntax-valtype>`, we define an auxiliary function to map such packed types into value types:
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \packed(\K{i8x16}) &=& \I32 \\
+   \packed(\K{i16x8}) &=& \I32 \\
+   \packed(t\K{x}N) &=& t \\
+   \end{array}
+
+
+.. _valid-vsunop:
+
+:math:`t\K{.}\vsunop`
+.....................
+
+* The instruction is valid with type :math:`[t] \to [t]`.
+
+.. math::
+   \frac{
+   }{
+     C \vdashinstr t\K{.}\vsunop : [t] \to [t]
```

Comment:
`t` is always `v128` in these rules, right? So there is no point in abstracting it away.